src/blacken_docs/__init__.py

```diff
@@ -13,10 +13,14 @@
 from black.mode import TargetVersion
 
 
-PYGMENTS_PY_LANGS = frozenset(("python", "py", "sage", "python3", "py3", "numpy"))
+PYGMENTS_PY_LANGS = frozenset(
+    ("python", "py", "sage", "python3", "py3", "numpy", ".py", ".python")
+)
 PYGMENTS_PY_LANGS_RE_FRAGMENT = f"({'|'.join(PYGMENTS_PY_LANGS)})"
```

Comment:
This is no longer precise since the `.` from `.py` is being added, and that will match any character, so `spy` would be allowed. `re.escape` could be used. But also I’m not sure extending the single markdown regex is the best approach, it might be more maintainable to have several, like the MyST PR approach.

Reply:
Now fixed.